src/vm/vm.c

```diff
@@ -796,6 +811,20 @@ static void setReplVar(DictuVM *vm, Value value) {
     tableSet(vm, &vm->globals, vm->replVar, value);
 }
 
+static void copyAnnotations(DictuVM *vm, ObjDict *superAnnotations, ObjDict *klassAnnotations) {
+    for (int i = 0; i <= superAnnotations->capacityMask; ++i) {
+        if (IS_EMPTY(superAnnotations->entries[i].key)) {
+            continue;
+        }
+
+        Value val = superAnnotations->entries[i].value;
+        push(vm, val);
+
+        dictSet(vm, klassAnnotations, superAnnotations->entries[i].key, val);
+        pop(vm);
```

Comment:
We shouldn't need to push to the stack here as they should already be referenced in the parent class, which means they will get greyed there